### DISCUSSION TITLE
local database in weasis viewer

### DISCUSSION THREAD

**Discussion by Avinash-c-gowda** [nroduit/Weasis#427] [Q&A]
Hi, @nroduit ; is that possible to maintain local database in this weasis viewer?

Thanks

**Comment by nroduit** [comment]
No, Weasis is not designed to work with DB and data persistence. This is deliberate in order to preserve data confidentiality on all workstations where Weasis is installed.

Weasis has a high level of implementation for interconnection with various archive systems (PACS, VNA...). New software to better manage this interoperability will be available before the end of the year (successor of weasis-pacs-connector).
